conventions/documentation/create.md

```diff
@@ -16,17 +17,62 @@ parent: Conventions
 
 ---
 
-## Creating from Create button
+## Create button types
 
-**Create from button (YAML only):**
+### Single Create Buttons
 
-*Some resource types can only be created with YAML. The first time viewing the YAML creation, the samples panel should be open by default.*
+*Single resource create buttons appear in the upper right of the page.*
+![Single create button](../images/create-button-single.png)
+
+---
+
+### Dropdown Create Buttons
+
+*If multiple resources or multiple methods (or both) can be created from a button, a dropdown is used.*
+
+**Multiple methods to create one object**
+- Name of resource is included on button.
+- Name of resource is not included in methods to create it.
+- Methods are recommended to start with “With” or “From”.
+- “From YAML” should appear last, when applicable.
+
+![Dropdown create button multimethod 2](../images/create-button-dropdown-multiMethod2.png)
+
+![Dropdown create button multimethod 1](../images/create-button-dropdown-multiMethod1.png)
+
+**Multiple objects created from one button**
+- Button just reads “Create”.
+- Names of resources are listed as options.
+
+![Dropdown create button multiresource](../images/create-button-dropdown-multiResource.png)
+
+**Multiple methods to create multiple objects**
+- Button just reads “Create”.
+- Groups are used to separate resource types, with the methods in each group.
+- A Separator component can appear between resource types.
+
+![Dropdown create button multiresourcemethod](../images/create-button-dropdown-multiResourceMethod.png)
+
+---
+
+### Create buttons in selection fields
+
+*Fields that allow the user to optionally create a resource where they would otherwise be selecting one have a create action as the first option in the dropdown, seperated by a divider.*
+![Create from selection](../images/create-button-from-selection.png)
+
+---
+
+## Creation with form/YAML
+
+### Creation via YAML only:
+
+*Some resource types can only be created with YAML. The first time viewing the YAML creation, the samples panel should be open by default. After the resource is created, the user is automatically navigated to the created resource.*
 ![import YAML](../images/create-yaml-only1.png)
 ![import YAML](../images/create-yaml-only2.png)
 
 ---
 
-**Create from button (Form only):**
+### Creation via form only:
 
 *Some resource types can only be created with a form. After the resource is created, the user is automatically navigated to the created resource.*
 ![import YAML](../images/create-form-only1.png)
```

Comment:
Same comment, since it's conventions, can we show this screen with the proper toolbar and button on the right. @gdoyle1 recently filed bugs for any pages like this that were still not converted.